exporter/clickhouseexporter/internal/exponential_histogram_metrics.go

```diff
@@ -18,7 +18,7 @@ import (
 const (
 	// language=ClickHouse SQL
 	createExpHistogramTableSQL = `
-CREATE TABLE IF NOT EXISTS %s_exponential_histogram %s (
+CREATE TABLE IF NOT EXISTS %s %s (
```

Comment:
This is great to see, glad we have control over the table name from the code instead of having the suffix baked into the SQL 👌